server.go

```diff
@@ -34,16 +44,19 @@ func main() {
 	// set up multiplexer.
 	router := http.NewServeMux()
 
+	// set up applyMiddleware
+	applyMiddleware := applyMiddlewareCreator(m.WithCORS)
+
 	// user management
-	router.HandleFunc("/user/get", d.HandleGetUser)
-	router.HandleFunc("/user/update", d.HandleUpdateUser)
-	router.HandleFunc("/user/create", d.HandleInitializeUser)
+	router.HandleFunc("/user/get", applyMiddleware(d.HandleGetUser))
+	router.HandleFunc("/user/update", applyMiddleware(d.HandleUpdateUser))
+	router.HandleFunc("/user/create", applyMiddleware(d.HandleInitializeUser))
```

Comment:
As mentioned earlier, I recommend that you simply wrap these with `m.WithCORS` for the time being. We do not yet need a wrapper function for large quantities of middleware.